src/mmw/js/src/modeling/views.js

```diff
@@ -131,6 +133,7 @@ var ProjectMenuView = Marionette.ItemView.extend({
 
         rename.on('update', function(val) {
             self.model.updateName(val);
+            self.model.saveProjectAndScenarios();
```

Comment:
Here are all the places where we are setting the name:

 * [`compare/controllers:copyProject`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/compare/controllers.js#L96): but it doesn't count since we also set `allow_save` to `false`, so we don't intend to save.
 * [`draw/controllers:enableSingleProjectModeIfActivity`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/draw/controllers.js#L69): but we're not so much setting the name as creating a new project, which would get saved when we enter the Analyze view as of #687 
 * [`modeling/controllers:makeProject`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/controllers.js#L264): same as above, where we will save if any changes are made to the project or scenarios after it has been made.

Thanks for pointing this out, glad I checked, but I think we're okay.